fix/49_LISTINGSEEDER

Symphonie234 opened the pull request:
### Changes Made:

- Created a new Seeder file named **`UserSeeder.php`**.
- Adjusted the code in the **`ListingSeeder.php`**.

Symphonie234 committed:
Fix for ListingSeeder bug

ChronoDevs commented on database/seeders/UserSeeder.php:
```diff
@@ -0,0 +1,53 @@
+    public function run()
+    {
+        // Create default users
+        User::create([
```
try to use $this->facker()

Symphonie234 committed:
attended to the requested changes in the PR No. 1

Symphonie234 commented:
### Changes that have been addressed in the PR No. 1:

- Adjusted the **`UserSeeder.php`** to utilize **Faker**.
- Changed the type of **zip_code** in the **`create_users_table`** table from **int()** to **string()** to handle different kinds/format of zip code.
